hazelcast/src/test/java/com/hazelcast/config/alto/AltoConfigTest.java

```diff
@@ -46,10 +47,27 @@ public void testEventloopCount() {
     }
 
     @Test
-    public void testConfigBounds() {
+    public void testConfigValidation() {
         AltoConfig altoConfig = config.getAltoConfig();
-        assertThrows(InvalidConfigurationException.class, () -> altoConfig.setEventloopCount(0));
-        assertThrows(InvalidConfigurationException.class, () -> altoConfig.setEventloopCount(256 + 1));
+        assertThrows(IllegalArgumentException.class, () -> altoConfig.setEventloopCount(0));
+    }
+
+    @Test
+    public void testSystemProperties() {
+        System.setProperty(ALTO_ENABLED.getName(), "true");
+        System.setProperty(ALTO_EVENTLOOP_COUNT.getName(), "3");
```

Comment:
These properties are not cleaned up upon the completion of the test. I think it can have an impact on other tests in this file in future

Reply:
Oh, that is nice. I didn't know about it